src/modapi/moditem.h

```diff
@@ -72,4 +73,56 @@ struct CModAPI_ModItem_LineStyle
 	int m_AnimationSpeed;
 };
 
+enum ESkinModifier
```

Comment:
Well, that would be also possible. The server would parse the json file of the skin we want, and set all parts according to that. Good idea.